src/Parse.java

```diff
@@ -52,9 +63,33 @@ public Parse(String filename, boolean parseNPrint) {
         }
         return payBytes;
     }
+
+    public static boolean hasCmdArgsYamlFlag (String args[]) {
+	return args.length >= 1 && args[0].equals("-y");
+    }
+
+    public static String readYamlPathArgs (String args[]) {
+	if (hasCmdArgsYamlFlag(args)) {
+	    return args[1];
+	} else {
+	    return "..//itch5.yaml";
+	}
+    }
+
+    public static String readItchPathArgs (String args[]) {
+	int argsPos = hasCmdArgsYamlFlag(args) ? 2 : 0;
+	if (args.length > argsPos) {
+	    return args[argsPos];
+	} else {
+	    return "";
+	}
+    }
+
     public static void main(String args[]) throws IOException, InterruptedException {
-        String filename = args[0];
-        Parse parse = new Parse(filename, true); // true to print otherwise enter false
+	String yamlPath = readYamlPathArgs(args);
+        String filename = readItchPathArgs(args);
```

Comment:
@Mister-Meeseeks I never trust the user. Command lines arguments placement can be judged using the file extensions don't you think.